Tests/TestingTests/Traits/ConditionTraitTests.swift

```diff
@@ -0,0 +1,45 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+@testable import Testing
+
+@Suite("Condition Trait Tests", .tags(.traitRelated))
+struct ConditionTraitTests {
+//  #if compiler(>=6.1)
+// TODO: Verify this will cause release/6.0 CI to fail
```

Comment:
Do you know how to run CI pipeline for swift-testing PR? @stmontgomery

I have verified this locally on macOS and Linux.

But I'd like to first verify the crash behavior on CI and then uncomment the `#if compile(>=6.1)` line to guard the test case.

Reply:
I have triggered CI, but I am fairly sure the PR-triggered CI runs against a newer development snapshot toolchain which is known to have the fix. I'm not sure we'll be able to validate the failure scenario in CI, we may just need to rely on your local testing. The change itself looks good to me (aside from needing to remove the commented-out `#if` guards

Reply:
Well, CI passed, which is basically expected. @Kyle-Ye I suggest you go ahead and remove these commented-out `#if` guards and we can proceed with landing this.

Reply:
Done